cmd/juju/backups/restore.go

```diff
@@ -76,6 +76,9 @@ https://jaas.ai/docs/controller-backups for more information.
 
 If the provided state cannot be restored, this command will fail with
 an explanation.
+
+WARNING: This command is deprecated in favor of the stand-alone
+"juju-restore" program: https://github.com/juju/juju-restore
```

Comment:
leave out "program". Or if an extra noun is really desired, maybe "utility" or "tool"

ideally we'd have "real" docs we'd point people to but the gh url is ok for now.

Reply:
Thanks. Changed program to tool (I think it reads a bit funny without the noun).

Agreed re real docs. We can put the docs link on the GH repo when they're ready and/or update these deprecation messages.